R/build-quarto-articles.R

```diff
@@ -1,5 +1,8 @@
 build_quarto_articles <- function(pkg = ".", article = NULL, quiet = TRUE) {
   pkg <- as_pkgdown(pkg)
+  if (pkg$bs_version < 5) {
```

Comment:
Do I also need to check for `NULL`?

Reply:
is it NULL at this stage if nothing is set in the config file?